Automattic/wp-calypso
# Redirect /themes/:theme to /theme/:theme for logged in users.

#### Proposed Changes

* Refactor the redirect implementation for /themes/:theme to also work for logged-in users.

#### Testing Instructions



* Use the Calypso live link
* Go to /themes/zaino and notice that you are now redirected to /theme/zaino
* Go to /themes/store and notice that you are still displayed the search page
* Repeat the steps for sites: /themes/zaino/:site

#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/issues/71033
